ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,201 @@
+
+Each ZEP must have a champion—someone who writes the ZEP using the style and format described below, shepherds the discussions in the appropriate forums, 
+and attempts to build community consensus around the idea. The ZEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is suitable for a ZEP. 
+Posting to the [Gitter](https://gitter.im/zarr-developers/community) is the best way to go about doing this.
+
+Vetting an idea publicly before going as far as writing a ZEP is meant to save the potential author time. Asking the Zarr community first if an idea is original 
+helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussion. It also helps to make sure the idea applies to 
+the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people.
+
+Once the champion has asked the Zarr community whether an idea has any chance of acceptance, a draft ZEP should be presented to the appropriate venue mentioned 
+above. This gives the author a chance to flesh out the draft ZEP to make properly formatted, of high quality, and to address initial concerns about the proposal.
+
+After the PR for the ZEP is in place, a post should be made on the Gitter channel containing the sections up to “Backward compatibility”, to limit discussion 
+there to usage and impact. Discussion on the pull request will have a broader scope, also including details of implementation.
+
+Standards Track ZEPs consist of two parts, a design document and a reference implementation. It is generally recommended that at least a prototype implementation 
+be co-developed with the ZEP, as ideas that sound good in principle sometimes turn out to be impractical when subjected to the test of implementation. 
+Often it makes sense for the prototype implementation to be made available as PR to the Zarr repo (making sure to appropriately mark the PR as a WIP).
+
+### Submitting a ZEP
+
+The proposal should be submitted as a draft ZEP via a GitHub pull request to the zarr-developers/zep directory with the name `zep-<n>.rst` where `<n>` is an appropriately assigned four-digit number (e.g., zep-0000.rst). The draft ZEP must use the [ZEP X - Template and Instructions](https://github.com/MSanKeys963/governance/blob/master/ZEP/template/template.md) file.
+
+A few points to consider while submitting your ZEP:
+
+- It should sound complete. The ideas must make technical sense.
+- The title should accurately describe the content.
+- The ZEP’s language (spelling, grammar, sentence structure etc.) and code style should be correct and conformant.
+
+The [Zarr Steering Council](https://github.com/zarr-developers/governance/blob/master/GOVERNANCE.md#steering-council) will not unreasonably deny publication of a ZEP. Reasons for denying ZEP include duplication of effort, being technically unsound, not providing proper 
```

Comment:
I'm not really convinced that this need be the ZSC. I can understand that it may simplify bootstrapping at this point, but I'd encourage others' ideas on a way to make this more inclusive.

Reply:
I think the reason to deny publication of ZEP is something on which ZSC could act swiftly. Deferring this decision to the community will only result in an extended timeframe.

Also, would love to hear the thoughts of others from ZSC.
Cc: @alimanfoo @jakirkham @rabernat @ryan-williams

Reply:
Why would we ever deny publication of a ZEP?

Reply:
Added Zarr Implementations Council too.